rust/datafusion/src/physical_plan/hash_aggregate.rs

```diff
@@ -329,6 +329,76 @@ fn group_aggregate_batch(
     Ok(accumulators)
 }
 
+/// Create a key `Vec<u8>` that is used as key for the hashmap
+fn create_key(group_by_keys: &[ArrayRef], row: usize, vec: &mut Vec<u8>) -> Result<()> {
```

Comment:
This moved to `hash_aggregate`. It can move once hash aggregate does stops using `Vec<u8>` as a key